src/Elcodi/Bundle/CouponBundle/Resources/config/doctrine/Coupon.orm.yml

```diff
@@ -49,6 +49,12 @@ Elcodi\Component\Coupon\Entity\Coupon:
             column: minimum_purchase_amount
             type: integer
             nullable: true
+        stackable:
+            column: stackable
+            type: boolean
+            nullable: false
+            options:
+                default: false
```

Comment:
Hmm, I see, this data is for existing entries? I saw that you've already set as false in Factory.